lib/src/builder/codegen.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2016 Workiva Inc.
```

Comment:
👍 

Ah, yeah, I forgot to mention: I'm going to do a follow-up PR after this one to update copyright notices, since they're missing in quite a few files